lib/environments/meteorEnvironment.js

```diff
@@ -1,22 +1,46 @@
+import fs from 'fs';
+import path from 'path';
+
+import findPackageDependencies from '../findPackageDependencies';
+
+const coreModules = [
+  'meteor/accounts-base',
+  'meteor/blaze',
+  'meteor/check',
+  'meteor/ddp-client',
+  'meteor/ddp-rate-limiter',
+  'meteor/ejson',
+  'meteor/email',
+  'meteor/http',
+  'meteor/check',
+  'meteor/meteor',
+  'meteor/mongo',
+  'meteor/random',
+  'meteor/reactive-var',
+  'meteor/session',
+  'meteor/templating',
+  'meteor/tracker',
+];
+
+function meteorPackageDependencies(
+  { config }: Object
+): Array<string> {
+  const meteorPackagesPath = path.join(config.workingDirectory, '.meteor/packages');
+  if (!fs.existsSync(meteorPackagesPath)) {
+    return [];
+  }
+  const packages = fs.readFileSync(meteorPackagesPath, 'utf8')
+    // extract an array of package names from the packages file
+    .match(/^\s*([^# @\n]+)/gm)
```

Comment:
Storybook has an `@` in its package name. https://github.com/kadirahq/react-storybook/blob/master/package.json#L2